src/strings/en-us.json

```diff
@@ -1722,5 +1722,6 @@
     "AiTranslated": "AI Translated",
     "MachineTranslated": "Machine Translated",
     "ForeignPartsOnly": "Forced/Foreign parts only",
-    "HearingImpairedShort": "HI/SDH"
+    "HearingImpairedShort": "HI/SDH",
+    "LabelIsHearingImpaired": "For hearing impaired (SDH)"
```

Comment:
It's nothing that you need to change here, but I've been letting people know that they will probably experience fewer conflicts in their PRs if they add new translations in alphabetical order instead of appending to the end of the file. :wink: